__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const request = require("supertest")
+
+const index = require("../db/data/test-data/index");
+
+beforeEach(() => {
+ return seed(index);
+});
+
+afterAll(() => {
+db.end();
+});
+
+describe.only("get /api/topics", () => {
+  test("responds with status 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns correct body", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((res) => {
+        console.log(">>>>TEST",res.body[0]);
+        const topic = res.body[0];
```

Comment:
We should be making an assertion for the amount of topics that come back. We can do this by checking the length of what we are returning back.

Reply:
following on from that, we should be iterating over each topic that has come back to check that it is structured correctly. We can do this by using the logic you have already but just placing it inside a forEach. Or we can use expect.ObjectContaining - see jest docs for this.